monitoring/mock_uss/scdsc/routes_injection.py

```diff
@@ -70,8 +77,21 @@ def query_operational_intents(
 @requires_scope([SCOPE_SCD_QUALIFIER_INJECT])
 def scdsc_injection_status() -> Tuple[str, int]:
     """Implements USS status in SCD automated testing injection API."""
-    return flask.jsonify({"status": "Ready", "version": versioning.get_code_version()})
+    return flask.jsonify({'status': 'Ready', 'version': versioning.get_code_version()})
+
+@webapp.route('/scdsc/v1/startreport', methods=['POST'])
```

Comment:
Let's make these endpoints [more RESTful](https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/#h-use-nouns-instead-of-verbs-in-endpoint-paths).  I think that would mean making the endpoint `/reports`, then POSTing to that endpoint would create a new report with a particular ID which could then be retrieved (GET by ID) or stopped (PUT/PATCH by ID).

Reply:
Will do.